api/v1/tag.go

```diff
@@ -176,7 +176,7 @@ func convertTagFromStore(tag *store.Tag) *Tag {
 	}
 }
 
-var tagRegexp = regexp.MustCompile(`#([^\s#,]+)`)
+var tagRegexp = regexp.MustCompile(`#((?:[^\s\p{P}]|_)+)`)
```

Comment:
It's recommended to handle potential errors when using `regexp.MustCompile`. Although it's unlikely that the regex will fail to compile, it's a good practice to handle this potential error. You can use `regexp.Compile` instead and handle the error. [important]